src/components/KonnectorTile.jsx

```diff
@@ -28,8 +40,7 @@ const KonnectorTile = ({
         />
       </header>
       <h3 className="item-title">{konnector.name}</h3>
-      {subtitle && <p className="item-subtitle">{subtitle}</p>}
-      {hasUserError ? svgIcon('warning') : svgIcon('check')}
+      {subPart}
```

Comment:
Previous implementation was using a inner component named `KonnectorTileFooter`. I suggest to use this name again, as it feels more explicit. What do you think ?